ebean-api/src/main/java/io/ebean/config/dbplatform/mariadb/MariaDbPlatform.java

```diff
@@ -11,6 +16,21 @@
   public MariaDbPlatform() {
     super();
     this.platform = Platform.MARIADB;
+    this.sequenceBatchMode = false;
     this.historySupport = new MariaDbHistorySupport();
   }
+  
+  @Override
+  protected void configureIdType(IdType idType) {
+    if (idType == IdType.SEQUENCE) {
+      this.dbIdentity.setSupportsSequence(true);
```

Comment:
Hmmm, I think we should do this on the constructor and not here.

Reply:
Our idea was, not to enable sequences by default. It has to be explicitly enabled in config:

If someone switches to a newer ebean versions, all migrations would use "SEQUENCE" from then on - and this makes it incompatible with MariaDB <10.3

Reply:
I feared that SEQUENCE is used if someone has used `GeneratedValue(strategy=GenerationType.SEQUENCE)`.
Old behaviour: AUTO_INCREMENT is used for mariadb, SEQUENCE will be used for other platforms
New behaviour: SEQUENCE will be used also for mariadb. The user has to update/migrate existing tables to SEQUENCE